vendor/bat-native-ledger/src/bat/ledger/internal/server/api/endpoints/get_parameters_unittest.cc

```diff
@@ -0,0 +1,290 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/test/task_environment.h"
+#include "bat/ledger/internal/ledger_client_mock.h"
+#include "bat/ledger/internal/ledger_impl_mock.h"
+#include "bat/ledger/internal/server/api/endpoints/get_parameters.h"
```

Comment:
Do we need so much folder nesting? One thing to note is that the caller really doesn't care about which servers are associated with each endpoint - it's the endpoint that matters.

A suggestion:

```cpp
#include "bat/ledger/internal/endpoints/api/get_parameters_endpoint.h"
```

In general:

```cpp
#include "bat/ledger/internal/endpoints/<server>/<something>_endpoint.h"
```